app/views/posts/index.html.erb

```diff
@@ -12,7 +12,27 @@
       <% end %>
     <% end %>
 
+    <h3>Comments</h3>
+      <% post.comments.each do |comment| %>
+    <p>
+      <strong>Comment:</strong>
+      <%= comment.body %>
+    </p>
+    <% end %>
+
+    <h2>Add a comment:</h2>
+      <%= form_for([post, post.comments.build]) do |comment| %>
+    <p>
+      <%= comment.label :body %><br>
```

Comment:
Do we need a big h2 title "Add a comment:" and a label "body"? Looks a bit funny